test/test_untilTag.js

```diff
@@ -0,0 +1,31 @@
+const fs = require('fs');
+const dcmjs = require('C:/Users/DHDESK/Desktop/dcmjs/dcmjs/build/dcmjs.js')
```

Comment:
Obviously this hard-coded path won't work for other people 😄 

Can you integrate this with the testing system so that it will run on any machine as part of `npm test`?  Have a look at the other test files for inspiration.

Reply:
Whoops, didn't mean to leave that in. Should be fixed now.